docs/jobs.md

```diff
@@ -39,11 +39,21 @@ import socket
 def add(x, y):
     job = get_current_job()
     job.meta['handled_by'] = socket.gethostname()
-    job.save()
+    job.update_meta()
+    
+    # do more work
+    time.sleep(1)
     return x + y
 {% endhighlight %}
 
-
+<div class="warning">
+    <img style="float: right; margin-right: -60px; margin-top: -38px" src="{{site.baseurl}}img/warning.png" />
+    <strong>Warning!</strong>
+    <p>
+The `.update_meta()` was introduced in RQ 0.7.2. You can use the `.save()` method instead 
+for RQ <= 0.7.1. For that case you also have to set the job `ttl` greater than the job `timeout`, otherwise the job may dissappear from the queue during execution.
```

Comment:
@selvin, this closed PR is probably is not the best place for discussion. Hovewer, the question is still opened for me, shouldn't I propose an another PR? I mean, is it worth modifying the code in the way I described above, i.e make all "cleanup" calls explicit? Thanks.